teradatalabs/cdh5-hive-kerberized/Dockerfile

```diff
@@ -69,6 +69,9 @@ ADD files/conf/hive-site.xml /etc/hive/conf/hive-site.xml
 RUN chown hive:hadoop /etc/hive/conf/hive.keytab
 RUN chmod 644 /etc/hive/conf/hive.keytab
 
+# ENABLE AUTHORIZATION IN HIVE SERVER 
+ADD files/conf/hiveserver2-site.xml /etc/hive/conf/hiveserver2-site.xml
```

Comment:
What does '2' in the file's name mean? Isn't this config strictly related to kerberos / hive authorization commands? If so, please express it in the name, e.g. 'hive-site-kerberos-auth-commands-config.xml`